src/speed_challenge_ctrl.cpp

```diff
@@ -32,14 +50,211 @@ class SpeedChallenge{
 
     ros::NodeHandle nh_;
     ros::NodeHandle private_nh_;
-    ros::Subscriber prop_sub;
 
-    prop_mapper::PropArray prop_array_test;
-    prop_mapper::PropArray prop_array;
+    ros::Subscriber prop_sub_;
+    ros::Subscriber task_to_exec_;
+    ros::Subscriber global_pos_;
+    ros::Publisher task_pos_;
+   
+    //Params 
+    float avoidance_angle;
```

Comment:
sorry this is new, but I think it'd be good to use "_p" at the end of parameter names